Orange/widgets/unsupervised/owkmeans.py

```diff
@@ -90,6 +90,7 @@ class OWKMeans(widget.OWWidget):
                   "quality estimation."
     icon = "icons/KMeans.svg"
     priority = 2100
+    keywords = ["means", "kmeans"]
```

Comment:
"means" is unnecessary as that worked before keywords already... The names are split into words so k-Means matched k and Means (and prefixes).
I would add "clustering" however.